packit/transformator.py

```diff
@@ -12,18 +12,18 @@
 import git
 
 from ogr.abstract import GitProject
-from sourcegit.config import PackageConfig
-from sourcegit.constants import DG_PR_COMMENT_KEY_SG_PR, DG_PR_COMMENT_KEY_SG_COMMIT
-from sourcegit.downstream_checks import get_check_by_name
-from sourcegit.utils import get_rev_list_kwargs, FedPKG, run_command
-from sourcegit.watcher import SourceGitCheckHelper
+from packit.config import PackageConfig
+from packit.constants import DG_PR_COMMENT_KEY_SG_PR, DG_PR_COMMENT_KEY_SG_COMMIT
+from packit.downstream_checks import get_check_by_name
+from packit.utils import get_rev_list_kwargs, FedPKG, run_command
+from packit.watcher import SourceGitCheckHelper
 
 logger = logging.getLogger(__name__)
 
 
 class Transformator:
     """
-    Describes relation between sourcegit and distgit repository.
+    Describes relation between packit and distgit repository.
```

Comment:
I am not sure if the original repo should not be called sourcegit.

Reply:
I blame pycharm!